### ISSUE TITLE
Way to handle suspended Collective applications

### ISSUE THREAD

**Issue by alanna** [opencollective/opencollective#6775]
## User story 
As a host admin, when processing Collective applications it's often the case that we go back for more information or changes and they don't respond, or they need more time, and the pending application sits there indefinitely. We can't then differentiate between active applications we need to review and old ones where we are waiting on them.

## Best solution for this problem
Either a status the admin can put the application in (e.g. when they first apply it's `pending` then it goes to `suspended` when the admin clicks a button or something), or a way the admin can reject the application and the Collective can later come back and reactivate it without needing to apply all over again. (It's kind of a similar situation to incomplete expenses in a way.)

**Comment by kaylarep** [comment]
OCF manages applications almost entirely in Slack & Freshdesk.  

This is for a few reasons (including not having a clear record of past messages-issues detailed hereL #4699 , having multiple team members who engage with applications, its easier to track pending status in FD, its easier to have internal conversations about applications in Slack). I'm not sure how other hosts manage applications _without_ FD/Slack.

Having this status option could be helpful for other fiscal host team members (who are not in the slack/FD channels) to know the latest updates with an application.  could even be nice to have an 'internal notes' section where we could provide a link to the FD thread or something.)

**Comment by alanna** [comment]
Having a comment thread on applications viewable to Host admins would be great.
